src/PatientBrowserTutorial/index.html

```diff
@@ -0,0 +1,208 @@
+<!DOCTYPE html>
```

Comment:
Where do you include `config.js` or `config.local.js`, are you sure this is tested?